packages/server/src/routes/members/index.ts

```diff
@@ -353,9 +401,24 @@ export const give = (
       return newOperationRef;
     });
 
+    const newOperationData = (await newOperationReference.get()).data();
```

Comment:
Do we need to wrap this await with the try as well?

Reply:
I think it would be a legit 500 if this failed, since we're getting the ref returned by the transaction which is creating the give operation.